Language-Version-History.md

```diff
@@ -1,8 +1,26 @@
 Features Added in C# Language Versions
 ====================
 
+# C# 10.0 - .NET 6 and Visual Studio 2022 version 17.0
+
+- [Record structs](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-10.0/record-structs.md) and `with` expressions on structs (`record struct Point(int X, int Y);`, `var newStructPoint = structPoint with { X = 100 };`).
```

Comment:
>var newStructPoint = structPoint with { X = 100 }; [](http://example.com/codeflow?start=183&length=50)

Consider using the exact example from the earlier "Records" item since it's shorter: `var newPoint = point with { X = 100 };`.